src/CSET/recipes/transect.yaml

```
@@ -1,4 +1,4 @@
-title: Transect
+title: Transect $CS_VARS
```

```suggestion
title: Transect of $CS_VARS
```
This makes it read better.